mmv1/products/bigquerybiglake/product.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: Biglake
+display_name: Biglake
```

Comment:
I'm wondering if these lines should be prefixing the product name with BigQuery, similar to what's present in [`products/bigqueryanalyticshub/product.yaml`](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/bigqueryanalyticshub/product.yaml), `/products/bigqueryconnection/product.yaml` and other BigQuery-related products.

This change would mean that the package for this service would match the pattern for other BigQuery-related services, and also would mean that the provider's documentation would be listed in the sidebar nav as part of this block of similarly-named categories:

<img width="362" alt="Screenshot 2023-08-29 at 12 55 53" src="https://github.com/GoogleCloudPlatform/magic-modules/assets/15078782/a9a2dbe4-5263-40a2-8453-0d31e60880b0">

Reply:
Discussed within the team and I'm going to change the folder name to `mmv1/products/biglake`